actinia/mapset.py

```diff
@@ -68,16 +69,43 @@ def __request_raster_layers(self):
 
     def get_raster_layers(self, force=False):
         """
-        Return raster layers
+        Return raster layers of the mapset
         """
         if self.raster_layers is None or force is True:
             self.__request_raster_layers()
         return self.raster_layers
 
+    def __request_vector_layers(self):
+        """
+        Requests the vector layers in the mapset.
+
+        :return: A list of the vector maps
```

Comment:
In GRASS, a single vector map can have several layers. Therefore I suggest to use vector maps here, and also raster maps instead of raster layers. Unfortunately, the actinia endpoints are called `raster_layers` and `vector_layers`.

Reply:
Ok 👌